packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -49,6 +49,7 @@ export interface ISiteProps {
     frameworks?: FrameworkEnum | FrameworkEnum[];
     activeFramework?: FrameworkEnum;
     collapsed?: boolean;
+    transparent?: boolean;
```

Comment:
I think the comment may have gotten lost somewhere but this should be more clear as to what is transparent, eg `componentBackgroundTransparent` like you have below.

Reply:
as this is a new prop, please also add to the documentation.